tools/coqdep/coqdep.ml

```diff
@@ -8,34 +8,34 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
-(** The basic parts of coqdep are in [Coqdep_common]. *)
-module CD = Coqdep_common
+(** The basic parts of coqdep are in [Common]. *)
+open Coqdeplib.Common
```

Comment:
In general I'm not in favor of using open like this, the global namespace is precious and we should avoid polluting it, specially when it was already prevented.

Reply:
That was from the other PR

Reply:
Yes I got confused by coqbot removing the needs rebase.